Header header
float32[] elements
sensor_msgs/CompressedImage RGBImage
